modules/apps/adaptive-media/adaptive-media-image-impl/src/main/java/com/liferay/adaptive/media/image/internal/util/RenderedImageUtil.java

```diff
@@ -90,7 +90,7 @@ public static RenderedImage readImage(InputStream inputStream)
 
 				return imageReader.read(0);
 			}
-			catch (IOException ioException) {
+			catch (Exception exception) {
```

Comment:
@robertoDiaz This will catch and ignore the ImageResolutionException we throw a couple of lines earlier, which looks wrong. Also, in the end an "unsupported image format" message will appear in the log, which will confuse users.

Is the ticket you mention in the comment (LPS-86924) correct? It looks unrelated.

Please resend ;-)